proto/v1/database_service.proto

```diff
@@ -62,6 +62,13 @@ service DatabaseService {
     option (google.api.http) = {get: "/v1/{name=instances/*/databases/*/schema}"};
   }
 
+  rpc DiffDatabaseSchema(GetDatabaseSchemaDiffRequest) returns (DatabaseSchemaDiff) {
+    option (google.api.http) = {
+      post: "/v1/{name=instances/*/databases/*/schema}:diff"
```

Comment:
Is `schema` some resource? Or is this a custom method, namely `diffSchema` on `database`?

What about `post: "/v1/{name=instances/*/databases/*}:diffSchema"`?

Reply:
ref https://google.aip.dev/136#stateless-methods

Reply:
`diffSchema` also okay for me

Reply:
I think it makes more sense to me to make this a stateless method.